toptal/picasso
# feat: add gradients

[FX-3799](https://toptal-core.atlassian.net/browse/FX-3799)

### Description

This pull request adds gradients.

As discussed with Matt Vella, for now it is enough to add 5 basic gradients from https://www.figma.com/file/I6n7U179JsKCGC57h6Jw0J/Toptal-Visual-Guidelines?node-id=2-1768&t=BGdqB1jYA8453SZz-0

### How to test

- Check out https://picasso.toptal.net/fx-3799-colors-add-color-gradients-to-picasso/?path=/story/utils-gradients--gradients

### Screenshots

https://user-images.githubusercontent.com/1390758/229873800-4bb0c8dd-e825-4ebd-847a-31e9535a53dd.mp4

### Development checks

- [N/A] Annotate all `props` in component with documentation

**Breaking change**

- [N/A] codemod is created and showcased in the changeset
- [N/A] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-3799]: https://toptal-core.atlassian.net/browse/FX-3799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
